cea/interfaces/cli/cli.config

```diff
@@ -35,7 +35,7 @@ sewage-potential = cea.technologies.sewage_heat_exchanger
 thermal-network-matrix = cea.technologies.thermal_network.thermal_network_matrix
 plots = cea.plots.plots_main
 network-layout = cea.technologies.thermal_network.network_layout.main
-decentralized = cea.optimization.preprocessing.disconnected_building_main
+decentralized = cea.optimization.preprocessing.decentralized_building_main
```

Comment:
i suggest we re-name the tool itself... how about decentralized-buildings-optimization

but what does "decentralized" even mean? are you talking about decentralized technologies like "air boxes"? or decentralized as in "not hooked up to the heating/cooling network? so... disconnected?

The naming has to mean as much as possible by itself - this is what will make the CEA be usable to other researchers.

Reply:
it is present in the interface under `Optimization` as decentralized-supply-system. So it will be redundant to have optimization again in the name.

Secondly, we have all agreed up on the nomenclature of using `decentralized` instead of `disconnected`, as we effectively do not have a building which is completely `disconnected` in an urban context (they will be linked to electrical grid at the minimum). Thus we are using the decentralized term. In academic research, there is still ambiguity about these two terms, but this is the description which we are going for (Also backed by good amount of literature)

Reply:
sure, the nomenclature tells that it is for decentralized buildings and also links to the supply system calculations

Reply:
hm... but... well, at our chair we have also used the term "decentralized supply systems" for things like airboxes... so, it's still confusing. this needs to be discussed more, i suggest we work on this in issue #1483 with its own pull request.